src/Workspaces/Core/Portable/Workspace/DocumentActiveContextChangedEventArgs.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis
+{
+    public sealed class DocumentActiveContextChangedEventArgs : EventArgs
+    {
+        public Workspace Workspace { get; }
+        public SourceTextContainer SourceTextContainer { get; }
+        public DocumentId PreviousContextDocumentId { get; }
+        public DocumentId NewContextDocumentId { get; }
```

Comment:
While we're talking about names, they should probably say ___**Active**ContextDocumentId, but yeah, opinions from @mattwar would be greatly appreciated on both of these points.